lib/ansible/modules/crypto/openssh_cert.py

```diff
@@ -0,0 +1,542 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, David Kainz <dkainz@mgit.at> <dave.jokain@gmx.at>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: openssh_cert
+author: "David Kainz (@lolcube)"
+version_added: "2.8"
+short_description: Generate OpenSSH host or user certificates.
+description:
+    - Generate and regenerate OpensSSH host or user certificates.
+requirements:
+    - "ssh-keygen"
+options:
+    state:
+        required: false
+        default: "present"
+        choices: [ 'present', 'absent' ]
+        description:
+            - Whether the host or user certificate should exist or not, taking action if the state is different from what is stated.
+    type:
+        required: true
+        choices: ['host', 'user']
+        description:
+            - Whether the module should generate a host or a user certificate.
+    force:
+        required: false
+        default: false
+        type: bool
+        description:
+            - Should the certificate be regenerated even if it already exists and is valid.
+    path:
+        required: true
+        type: path
+        description:
+            - Path of the file containing the certificate.
+    signing_key:
+        required: true
+        type: path
+        description:
+            - The path to the private openssh key that is used for signing the public key in order to generate the certificate.
+    public_key:
+        required: true
+        type: path
+        description:
+            - The path to the public key that will be signed with the signing key in order to generate the certificate.
+    valid_from:
+        required: true
+        type: str
+        description:
+            - "The point in time the certificate is valid from. Time can be specified either as relative time or as absolute timestamp.
+               Time will always be interpreted as UTC. Valid formats are: [+-]timespec | YYYY:MM:DD | YYYY:MM:DD:HH:MM:SS | YYYY:MM:DD HH:MM:SS | always
+               where timespec can be an integer + [w | d | h | m | s] (e.g. +32w1d2h).
+               Note that if using relative time this module is NOT idempotent."
+    valid_to:
+        required: true
+        type: str
+        description:
+            - "The point in time the certificate is valid to. Time can be specified either as relative time or as absolute timestamp.
+               Time will always be interpreted as UTC. Valid formats are: [+-]timespec | YYYY:MM:DD | YYYY:MM:DD:HH:MM:SS | YYYY:MM:DD HH:MM:SS | forever
```

Comment:
I think `C()` should help with formatting and readability

```suggestion
               Time will always be interpreted as UTC. Valid formats are: C([+-]timespec | YYYY:MM:DD | YYYY:MM:DD:HH:MM:SS | YYYY:MM:DD HH:MM:SS | forever)
```